tests/gui/TestGui.cpp

```diff
@@ -108,9 +97,23 @@ void TestGui::initTestCase()
     m_mainWindow->resize(1024, 768);
 }
 
-// Every test starts with opening the temp database
+// Every test starts with reseting config settings and opening the temp database
 void TestGui::init()
 {
+    // Disable autosave so we can test the modified file indicator
+    config()->set(Config::AutoSaveAfterEveryChange, false);
+    config()->set(Config::AutoSaveOnExit, false);
+    config()->set(Config::AutoReloadOnChange, true);
```

Comment:
Adding config()->set(Config::AutoReloadOnChange, true) in init() causes the autoreloadDatabase test to fail.